src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -747,63 +754,78 @@ void shouldThrowInvalidRoleAssignmentExceptionWhenCaseTypeIsInvalidAndJurisdicti
+        HearingEntity hearingEntity = generateHearingEntity(hearingId, DeleteHearingStatus.UPDATE_REQUESTED.name(), 1);
+        when(hearingRepository.findById(hearingId)).thenReturn(Optional.of(hearingEntity));
+        when(hearingRepository.save(hearingEntity)).thenReturn(hearingEntity);
+        when(caseHearingRequestRepository.getCaseHearing(hearingId)).thenReturn(entity);
+
+        HearingResponse hearingResponse = hearingManagementService.deleteHearingRequest(
+                hearingId, TestingUtil.deleteHearingRequest());
+        assertEquals(versionNumber + 1, hearingResponse.getVersionNumber());
+        assertEquals(CANCELLATION_REQUESTED, hearingResponse.getStatus());
+        assertNotNull(hearingResponse.getHearingRequestId());
+        verify(hearingRepository).existsById(hearingId);
+        verify(caseHearingRequestRepository).getVersionNumber(hearingId);
     }
 
     @Test
     void deleteHearingRequestShouldPassWithValidStatus() {
-        when(caseHearingRequestRepository.getVersionNumber(2000000000L)).thenReturn(1);
-        when(hearingRepository.existsById(2000000000L)).thenReturn(true);
-        when(hearingRepository.getStatus(2000000000L)).thenReturn("UPDATE_SUBMITTED");
-        when(hearingRepository.findById(2000000000L)).thenReturn(Optional.of(TestingUtil.deleteHearingEntity()));
-        hearingManagementService.deleteHearingRequest(2000000000L, TestingUtil.deleteHearingRequest());
-        verify(hearingRepository).existsById(2000000000L);
-        verify(caseHearingRequestRepository).getVersionNumber(2000000000L);
-    }
-
-    @Test
-    void deleteHearingRequestShouldFailWithInValidStatus() {
-        when(caseHearingRequestRepository.getVersionNumber(2000000000L)).thenReturn(1);
-        when(hearingRepository.existsById(2000000000L)).thenReturn(true);
-        when(hearingRepository.getStatus(2000000000L)).thenReturn("UPDATE_NOT_SUBMITTED");
-        Exception exception = assertThrows(BadRequestException.class, () -> {
-            hearingManagementService.deleteHearingRequest(2000000000L, TestingUtil.deleteHearingRequest());
-        });
+        final long hearingId = 2000000000L;
+        HearingEntity hearingEntity = generateHearingEntity(hearingId, DeleteHearingStatus.UPDATE_REQUESTED.name(), 1);
+        when(caseHearingRequestRepository.getVersionNumber(hearingId)).thenReturn(1);
+        when(hearingRepository.existsById(hearingId)).thenReturn(true);
+        when(hearingRepository.getStatus(hearingId)).thenReturn("UPDATE_SUBMITTED");
+        when(hearingRepository.findById(hearingId)).thenReturn(Optional.of(hearingEntity));
+        when(hearingRepository.save(hearingEntity)).thenReturn(hearingEntity);
+        HearingResponse hearingResponse = hearingManagementService.deleteHearingRequest(
+                hearingId, TestingUtil.deleteHearingRequest());
+        assertEquals(CANCELLATION_REQUESTED, hearingResponse.getStatus());
```

Comment:
can you add assertEquals(versionNumber + 1, hearingResponse.getVersionNumber());

Reply:
Done